openedx_blockstore_relay/tests/README.rst

```diff
@@ -0,0 +1,6 @@
+openedx-blockstore-relay Tests
+==============================
+
+Run these tests from the devstack CMS shell (``make studio-shell``) using::
+
+    python -m pytest --ds=cms.envs.test /edx/src/openedx-blockstore-relay/openedx_blockstore_relay/tests/
```

Comment:
We really need it on CI for a number of repos. Do you think it is feasible?

Reply:
It would be feasible if edX published docker images of `edx-lms` and `edx-studio` that (combined with standard `mysql`, `memcached`, etc. images) run easily. Then it would simply be a matter of configuring CI to run the `edx-studio` image, install this app into that image's virtualenv, then run the tests.

I believe the main work involved is getting the tooling in place to auto-publish working `edx-studio` container to Docker Hub. They only currently offer a devstack `edxapp` image, which is not self-contained.